yamllint: Enable empty-lines and trailing-spaces rules

aman556 opened the pull request:
## What this PR does / why we need it
<!--
Add a detailed explanation of what this PR does and why it is needed.
-->
- This is a follow up PR for [#1304](https://github.com/vmware-tanzu/community-edition/pull/1304).
- Enable empty-lines and trailing-spaces rules in yamllint configuration
- Fix all the empty-lines and trailing-spaces errors in yaml files

## Details for the Release Notes (PLEASE PROVIDE)
<!--
Unless this is a trivial change, we want to know more about your contribution!
This can even be a TLDR version of the "What this PR does".
If a trivial change, just write "NONE" in the release-note block below.
Otherwise, a release note is required:
-->
```release-note 
yamllint: Enable empty-lines and trailing-spaces rules; Clean all yaml files. 
```
## Which issue(s) this PR fixes
<!--
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes:  #1173
## Describe testing done for PR
<!--
Example: Created vSphere workload cluster to verify change. 
-->

- Run `make yamllint` and ensure no errors are returned by yamllint and it exits with 0

## Special notes for your reviewer
<!--
Add any things that reviewers should be aware of as they review
your PR.

Example: Please verify how I handled foo aligns with overall plan.
-->
This PR contain correction of trailing spaces and empty-lines warnings in all the YAML files. yamllint is enabled in #1304, this followup enables more rules and fixes all the respective yaml files.

stmcginnis reviewed (commented):
This is adding a bunch of new multus files. Is that a merge error?

aman556 committed:
yamllint: Enable empty-lines and trailing-spaces rules

- Enable empty-lines and trailing-spaces rules in yamllint configuration
- Fix all the empty-lines and trailing-spaces errors in yaml files

Signed-off-by: Aman Sharma <amansh@vmware.com>

stmcginnis reviewed (dismissed):
Looks like everything is covered, and nice clean up. Thanks!

Commit:
fix missing new line in grafana package

Signed-off-by: joshrosso <rossoj@vmware.com>

Commit:
Merge branch 'main' into yaml_linter_trailing-spaces_resolved
